app/controllers/films_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FilmsController < ApplicationController
+  def index
+    @films = helpers.get_csv_data
```

Comment:
Limit this to like 20/30/40 or 50 items. Parsing all the 8k csv rows takes a lot of time and it will create massive list. in the 2nd issue, there is a TIP saying:
`There are a lot of shows in the [provided CSV file](https://www.dropbox.com/s/iqqbb64z8eogqxb/netflix_titles.zip?dl=0)! You may need to limit the number you render to the view.`